environment.yml

```diff
@@ -0,0 +1,266 @@
+name: enzyhtp
+channels:
+  - salilab
```

Comment:
what is this for?

Reply:
This is the channel to install `modeller` package. See https://anaconda.org/salilab/modeller